cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -318,6 +337,28 @@ protected ClientConfig getClientConfig() {
     return builder.build();
   }
 
+  protected ClientConfig getClientConfig(@Nullable AccessToken accessToken) {
```

Comment:
should remove a lot of the duplicate code from the versions of these methods that are parameter-less